k2/csrc/util.h

```diff
@@ -31,8 +33,8 @@ struct PairHash {
   }
 };
 
-static const double kMinLogDiffDouble = Log(DBL_EPSILON);  // negative!
-static const float kMinLogDiffFloat = Log(FLT_EPSILON);  // negative!
+static const double kMinLogDiffDouble = log(DBL_EPSILON);  // negative!
+static const float kMinLogDiffFloat = log(FLT_EPSILON);    // negative!
```

Comment:
There is a function that is specific to `float`

```cpp
float logf(float x);
```

Reply:
Done